cesium_app/tests/frontend/test_build_model.py

```diff
@@ -37,6 +70,38 @@ def test_add_model(driver):
             driver.save_screenshot("/tmp/models_fail.png")
             raise
 
+
+def test_build_model_lr(driver):
+    driver.get('/')
+    with create_test_project() as p, create_test_featureset(p, type='regr') as fs:
+        driver.refresh()
```

Comment:
There's lots of duplicated code between these functions---refactor to _private_function.